### PR TITLE
Update to v1.0.0 and add QBO analysis and STE lat-lon plots 

### PR DESCRIPTION
**Pull request by hsiangheleellnl**
Two major chnages updated in this merge:
1. QBO analysis: including three types of plots. 
2. O3 STE flux monthly lat-lon plots. 

Other minor updates:
1. ChemDyg inputs on Zenodo and LCRC share folders
2. Example run_script including QBO option
3. Version updates to v1.0.0

### CODE REVIEW

**Comment by hsiangheleellnl** [comment]
@tangq Please review the pull request. Thank you!

File: chemdyg/templates/chemdyg_STE_flux_diags.bash [hunk 1]
```diff
@@ -71,30 +69,9 @@ do
...
-#if [ ${#files[@]} -ne 0 ]
-#then
-#  ln -s ${files[0]} .
-#fi
-#{%- endraw %}
-#{%- endif %}
-
-#cd ..
+mapDir="{{ diagnostics_base_path }}/observations/Atm/ChemDyg_inputs"
+ln -s ${mapDir}/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc ./
```
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash]: This map file only works for ne30pg2 files. Should it be a cfg input, so users can specify the mapping files by themselves?

File: chemdyg/templates/chemdyg_STE_flux_diags.bash [hunk 2]
```diff
@@ -234,6 +218,17 @@ else:
      yS_ann[0] = 'nan'
      yS_ann[1:12] = y[0:11]*1.E-9*12
 
+STE_month = xr.DataArray(np.array(STE_2D), coords=[month[1::],ncol], dims=["month","ncol"])
+STE_month_mean= np.zeros((12,len(ncol)))
+for i in range(12):
+    if len(time) == 12:
+        STE_month_mean[0,:] = 0
+        STE_month_mean[1::,:] = 1.e3*12*STE_month
```
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: Multiplying by 12 ignores that different months have different days. Same for other lines, like line 219 above and other places. Should fix this. Maybe in a separate PR after this one is merged.

File: chemdyg/templates/chemdyg_STE_flux_diags.bash [hunk 3]
```diff
@@ -305,6 +304,56 @@ line2 = 'NH mean:'+str(np.round(yN_mean,2))
 line3 = 'SH mean:'+str(np.round(yS_mean,2))
 plt.legend( [line1,'Global std',line2,'NH std',line3,'SH std'])
 pylab.savefig(pathout+'O3_STE_flux_monthly.png', dpi=300)
+
+#----- 12 month lat-lon plot---------
+cmd = "ncks --map=map_ne30pg2_to_cmip6_180x360_aave.20200201.nc E3SM_O3_STE_month_mean.nc E3SM_O3_STE_month_latlon.nc"
```
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash]: Hard-coded mapping file name will give us trouble. Make it a cfg input.

File: setup.cfg [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = chemdyg
-version = 0.1.5
+version = 1.0.0
```
Comment (tangq) [inline setup.cfg:3]: Good - the version number update was not forgotten.

**Review by tangq — CHANGES_REQUESTED** [review]

File: chemdyg/templates/chemdyg_STE_flux_diags.bash (hunk 2 above)
Comment (hsiangheleellnl) [reply chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: @tangq Different days in different months have been considered in earlier
```dt = monthrange(2001,month[i])[1]*3600*24```
*12 is to covert the unit from per month to per year.

Comment (tangq) [reply chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: Good to know it's been calculated. Would you add some comments to make it clearer?

**Comment by hsiangheleellnl** [comment]
@tangq The changes have been made as recommended. Please review and approve the pull request. Thanks.

**Review by tangq — APPROVED** [review]
I approve this PR if the test is successful.

**Comment by tangq** [comment]
@hsiangheleellnl , just a reminder - we will do a new release for this.

**Pull request merged**